CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 6.15.1
```

Comment:
this is new functionality, not a bug fix, so you will want to bump the minor version to `6.16.0`